docs/source/migration_guides/coming_from_dask.md

```diff
@@ -0,0 +1,122 @@
+
+```
+# apply a custom function “crop_image” to the image column
+@daft.udf(...)
+def crop_image(**kwargs):
+    …
+    return …
+
+df = df.with_column(
+    "cropped",
+    crop_image(daft.col(“image”), **kwargs),
+)
+```
+
+## Daft is built for Machine Learning Workloads
+
+Dask offers some distributed Machine Learning functionality through the [`dask-ml` library](https://ml.dask.org/). This library provides parallel implementations of a few common scikit-learn algorithms. Note that `dask-ml` is not a core Dask library and is not as actively maintained. It also does not offer support for deep-learning algorithms or neural networks.
+
+Daft is built as a DataFrame API for distributed Machine learning. You can use Daft UDFs to apply Machine Learning tasks to the data stored in your Daft DataFrame, including deep learning algorithms from libraries like PyTorch. See [our Quickstart](https://www.getdaft.io/projects/docs/en/latest/10-min.html) for a toy example.
+
+## Daft supports Multimodal Data Types
+
+Dask supports the same data types as pandas. Daft is built to support many more data types, including Images, nested JSON, tensors, etc. See [the documentation](https://www.getdaft.io/projects/docs/en/latest/user_guide/daft_in_depth/datatypes.html) for a list of all supported data types.
+
+## Distributed Computing and Remote Clusters
+
+Both Dask and Daft support distributed computing on remote clusters. In Dask, you create a Dask cluster either locally or remotely and perform computations in parallel there. Currently, Daft supports distributed cluster computing [with Ray](/user_guide/poweruser/scaling-up.html). Support for running Daft computations on Dask clusters is on the roadmap.
+
+Cloud support for both Dask and Daft is the same.
+
+## SQL Support
+
+Dask does not natively provide full support for running SQL queries. You can use pandas-like code to write SQL-equivalent queries, or use the external [`dask-sql` library](https://dask-sql.readthedocs.io/en/latest/).
+
+Daft provides a read_sql method to read SQL queries into a DataFrame. Daft uses SQLGlot to build SQL queries, so it supports all databases that SQLGlot supports. Daft pushes down operations such as filtering, projections, and limits into the SQL query when possible. Full-featured support for SQL queries (as opposed to a DataFrame API) is in progress.
+
+## Daft combines Python with Rust and Pyarrow for optimal performance
+
+Daft combines Python with Rust and Pyarrow for optimal performance. Under the hood, Table and Series are implemented in Rust on top of the Apache Arrow specification (using the Rust arrow2 library). This architecture means that all the computationally expensive operations on Table and Series are performed in Rust, and can be heavily optimized for raw speed. Python is most useful as a user-facing API layer for ease of use and an interactive data science user experience. Read [more](/faq/technical_architecture.rst).
```

Comment:
We could also link / embed the benchmarks here!

https://www.getdaft.io/projects/docs/en/latest/faq/benchmarks.html